packages/authentication/index.js

```diff
@@ -3,5 +3,6 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+/* eslint-disable no-unused-vars*/
 const nodeMajorVersion = +process.versions.node.split('.')[0];
```

Comment:
I think we should remove `nodeMajorVersion` if it's not used (instead of adding `eslit-disable` comment).

The same comment applies to all other places below where you added `eslint-disable`.

(BTW I believe we don't have `eslint` configured in our monorepo, do you have `eslint` configured in your IDE independently of our project setup?)

Reply:
Okay, will do. Yeah I have `eslint`, but realized we use `tslint` after, and I have that  change in the other PRs :).